app/Services/ContestManager.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Gladiator\Services;
+
+use Gladiator\Repositories\UserRepositoryContract;
+
+class ContestManager
+{
+    /**
+     * UserRepositoryContract instance.
+     *
+     * @var \Gladiator\Repositories\UserRepositoryContract
+     */
+    protected $repository;
+
+    /**
+     * Create new Registrar instance.
+     *
+     * @param Northstar $northstar
+     */
+    public function __construct(UserRepositoryContract $repository)
+    {
+        $this->repository = $repository;
+    }
+
+    public function exportCSV($model, $includeReportbackData = false)
```

Comment:
Add some docblocks and could probs just name `$includeReportbackData = false` -> `$reportbacks = false`